src/main/java/org/roda_project/commons_ip2/validator/common/ZipManager.java

```diff
@@ -106,7 +106,7 @@ public boolean checkIfExistsRootMetsFile(Path path) throws IOException {
     Enumeration entries = zipFile.entries();
     while (entries.hasMoreElements()) {
       ZipEntry entry = (ZipEntry) entries.nextElement();
-      if (entry.getName().matches(".*/METS.xml")) {
+      if (entry.getName().endsWith("METS.xml")) {
```

Comment:
Not considering "otherMETS.xml"
```suggestion
      if (entry.getName().endsWith("/METS.xml")) {
```